project/src/test/java/com/google/sps/data/PlaceTest.java

```diff
@@ -17,7 +17,10 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+
 import com.google.maps.model.LatLng;
+import com.google.common.collect.ImmutableList;
+
```

Comment:
I think the static imports should be before the non static ones

Reply:
Right